volumina/sliceSelectorHud.py

```diff
@@ -502,11 +617,18 @@ def _handlePositionBoxValueChanged(self, axis, value):
         new_position[changed_axis] = value
         self.positionChanged.emit(*new_position)
 
-    def updateShape5D(self, shape5D):
-        self.timeSpinBox.setMaximum(shape5D[0]-1)
-        self.xSpinBox.setMaximum(shape5D[1]-1)
-        self.ySpinBox.setMaximum(shape5D[2]-1)
-        self.zSpinBox.setMaximum(shape5D[3]-1)
+    def updateShape5D(self, shape5Dmax, shape5DcropMin):
```

Comment:
I'd say this function should be renamed now that it takes more than just a shape.  Also, a subtle point: most functions that take a roi take the args in (min, max) order, not (max, min) order.  It will be easier to understand if we stick with that convention.